lib/lndclient/LndClient.ts

```diff
@@ -780,7 +780,8 @@ class LndClient extends SwapClient {
       await this.connectPeerAddresses(uris);
     }
 
-    await this.openChannelSync(remoteIdentifier, units, pushUnits);
+    const openResponse = await this.openChannelSync(remoteIdentifier, units, pushUnits);
+    return openResponse.hasFundingTxidStr() ? openResponse.getFundingTxidStr() : base64ToHex(openResponse.getFundingTxidBytes_asB64());
```

Comment:
Do we need this ternary statement here? I think just `return base64ToHex(openResponse.getFundingTxidBytes_asB64())` is enough to keep things simple. Might be worth tidying up but I don't think it's necessarily worth holding up the PR.

Reply:
According to the documentation this's the field that actually should have transaction id, so I thought it would worth to check it with this statement.

Reply:
Which documentation?